lib/testing.sh

```diff
@@ -0,0 +1,39 @@
+#!/sur/bin/env bash
```

Comment:
I think it should be #!/usr/bin/env

Reply:
I also think we need the Kubernetes copyright at the top

Reply:
> I also think we need the Kubernetes copyright at the top

I agree, however we have no check for that yet and many other files miss that header. That's probably another thing we need to tackle somewhen, but I don't want to do it all in this PR. I however added the header to files this PR touches / files which are related to this PR.